mmdet/models/dense_heads/deformable_detr_head.py

```diff
@@ -96,63 +89,12 @@ def init_weights(self) -> None:
             for m in self.reg_branches:
                 nn.init.constant_(m[-1].bias.data[2:], 0.0)
 
-    def forward(self, x: Tuple[Tensor],
-                batch_img_metas: List[dict]) -> Tuple[Tensor, ...]:
+    def forward(self, hs: Tensor, init_reference,
```

Comment:
The name `hs` and `init_reference` may be confusing

Reply:
In the revised commit, I use the name `hidden_states` uniformly for the args interface.
The `init_reference` and `inter_reference` are going to be merged into a common arg.